src/ft/mod.rs

```diff
@@ -155,6 +164,11 @@ impl Rasterize for FreeTypeRasterizer {
     }
 
     fn load_font(&mut self, desc: &FontDesc, size: Size) -> Result<FontKey, Error> {
+        if self.creation_timestamp.map_or(true, |t| t.elapsed() > RELOAD_DELAY) {
```

Comment:
If we don't use `take` here, I think we can fit in a `timestamp` for the variable in the closure without exceeding the max column count per line. That way it's a little nicer to read.